packages/core/saved-objects/core-saved-objects-api-server/src/apis/find.ts

```diff
@@ -56,9 +56,25 @@ export interface SavedObjectsFindOptions {
   /** sort order, ascending or descending */
   sortOrder?: SortOrder;
   /**
-   * An array of fields to include in the results
+   * An array of attributes to fetch and include in the results. If unspecified, all attributes will be fetched.
+   *
+   * The main purpose of this option is to avoid fetching unnecessary heavy fields (e.g blobs) when searching for
+   * savedObjects, for performance purposes.
+   *
+   * Defaults to `undefined` (fetching all fields).
+   *
    * @example
-   * SavedObjects.find({type: 'dashboard', fields: ['attributes.name', 'attributes.location']})
```

Comment:
Fun fact: the example was wrong. fields shouldn't be prefixed by `attributes.`

Reply:
😱